app/Http/Controllers/V2/AuditStatus/GetAuditStatusController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers\V2\AuditStatus;
+
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\AuditStatusResource;
+use App\Models\V2\EntityModel;
+use Illuminate\Http\Request;
+
+class GetAuditStatusController extends Controller
+{
+    public function __invoke(Request $request, EntityModel $entity)
```

Comment:
Are all EntityModels auditable? I'd been proposing having a new interface to limit what can be passed into this controller to just those types of models that can be audited. Also, it looks from other parts of this PR that `SitePolygon` can be audited, and it doesn't implement `EntityModel`.